### PR TITLE
installed rubocop and atom plugins for rubocop and reek

### PR DESCRIPTION
**Pull request by DmitryGolubko**
Installing rubocop and reek 
Already merged with with installing-webpacker <- Circle-CI <- rubocop-config-changes branches
Changed base to master
Deleted previous created-new-app <- installing-rspec-capybara <- setting - up-.env-variables branches

### CODE REVIEW

**Comment by DmitryGolubko** [comment]
`reek` has been installed as separate gem and I can't find how to include it in project structure

**Comment by AleksSenkou** [comment]
@DmitryGolubko why didn't you ask about it in slack's `#general` channel?
Initiate discussion there. We'll find a way to setup everything rightly 😉

File: app/helpers/welcome_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
+end
```
Comment (AleksSenkou) [inline app/helpers/welcome_helper.rb]: @DmitryGolubko let's not keep files that don't implement logic. It's auto-generated 'empty' file

File: app/views/layouts/mailer.html.slim [hunk 2]
```diff
@@ -0,0 +1,7 @@
+doctype html
+html
+  head
+    meta content=("text/html; charset=utf-8") http-equiv="Content-Type" /
+    css:
```
Comment (AleksSenkou) [inline app/views/layouts/mailer.html.slim]: @DmitryGolubko why is this `css:` needed? I guess nothing will be changed if we remove it 😉 
Let's configure mail layout when we'll have mails

File: config/database.yml [hunk 3]
```diff
@@ -7,12 +7,22 @@ development:
   <<: *default
   database: budget-planning-app_development
 
+  host: localhost
+  port: 5432
+  username: budget_planning_app
+  password: 27051965
```
Comment (AleksSenkou) [inline config/database.yml]: @DmitryGolubko let's move password to env variable. `BUDGET-PLANNING-APP_DATABASE_PASSWORD` can be used. you'll have one value in development and another in prod

File: spec/controllers/welcome_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "routes", type: :routing do
+  describe 'Welcome #show' do
+    it 'routes to #show' do
+      expect(:get => "/").
+      to route_to(:controller => "welcome", :action => "show")
```
Comment (AleksSenkou) [inline spec/controllers/welcome_controller_spec.rb]: @DmitryGolubko can't be done in one line?
```ruby
expect(get: "/").to route_to(controller: "welcome", action: "show")
```

**Comment by AleksSenkou** [comment]
@DmitryGolubko before merge - be sure to check that CI is green. Currently, it's 🔴

File: app/helpers/welcome_helper.rb (hunk 1 above)
Comment (DmitryGolubko) [reply app/helpers/welcome_helper.rb]: Will it have affect on application execution in future?

Comment (AleksSenkou) [reply app/helpers/welcome_helper.rb]: @DmitryGolubko it won't affect an execution
but such files will make the app broader without a need

Comment (AleksSenkou) [reply app/helpers/welcome_helper.rb]: @DmitryGolubko keeping the app consistent is a goal from the begging

Comment (DmitryGolubko) [reply app/helpers/welcome_helper.rb]: OK :+1: I'm in process

Comment (DmitryGolubko) [reply app/helpers/welcome_helper.rb]: What do you think about empty directories?

File: app/views/layouts/mailer.html.slim (hunk 2 above)
Comment (DmitryGolubko) [reply app/views/layouts/mailer.html.slim]: just remove `mailer.html.slim` and `mailer.text.slim`?

File: config/database.yml (hunk 3 above)
Comment (DmitryGolubko) [reply config/database.yml]: Done !

File: spec/controllers/welcome_controller_spec.rb (hunk 4 above)
Comment (DmitryGolubko) [reply spec/controllers/welcome_controller_spec.rb]: Done!

File: app/views/layouts/mailer.html.slim (hunk 2 above)
Comment (DmitryGolubko) [reply app/views/layouts/mailer.html.slim]: Fixed)

File: app/helpers/welcome_helper.rb (hunk 1 above)
Comment (AleksSenkou) [reply app/helpers/welcome_helper.rb]: @DmitryGolubko I guess there is no need for keeping empty directories :)

**Pull request merged**